### PR TITLE
Automated cherry pick of #2994: Allow operations on felixconfigurations in tigera

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #2994 on release-v1.30.

#2994: Allow operations on felixconfigurations in tigera

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
